blueocean-config/src/test/java/io/jenkins/blueocean/config/FeaturesTest.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.blueocean.config;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class FeaturesTest {
+    @Test
+    public void testLookupFeature() {
+        Features features = new Features();
+        assertEquals("false", features.getFeature("my.cool.feature"));
+    }
+
+    @Test
+    public void testSystemPropertyOverride() {
+        Features features = new Features();
+        System.setProperty("feature.my.cool.feature", "true");
```

Comment:
Ah, this is overwriting my.cool.feature property to true and so the other test fails as it finds it true. setProperty() returns existing value (if present) that test could have set in finally block but it won't work since the property name  in internal map is not 'feature.my.cool.feature' but my.cool.feature. I think at least wee should either keep 'feature.' in all places or keep the names same al places.